# for debugging, so far it's enough

uint8 target_position
float32 target_distance